bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/internal/ThingManagerImpl.java

```diff
@@ -571,7 +570,23 @@ public void thingUpdated(final Thing thing, ThingTrackerEvent thingTrackerEvent)
                             oldThing.setHandler(null);
                         }
                         thing.setHandler(thingHandler);
-                        safeCaller.create(thingHandler, ThingHandler.class).build().thingUpdated(thing);
+
+                        if (isInitializable(thing, getThingType(thing))) {
+                            safeCaller.create(thingHandler, ThingHandler.class).build().thingUpdated(thing);
+                        } else {
+                            final ThingHandlerFactory thingHandlerFactory = findThingHandlerFactory(
+                                    thing.getThingTypeUID());
+                            if (thingHandlerFactory != null) {
+                                if (isBridge(thing)) {
+                                    unregisterAndDisposeChildHandlers((Bridge) thing, thingHandlerFactory);
+                                }
+                                disposeHandler(thing, thingHandler);
+                                setThingStatus(thing,
+                                        buildStatusInfo(ThingStatus.UNINITIALIZED,
+                                                ThingStatusDetail.HANDLER_CONFIGURATION_PENDING,
+                                                "Missing or invalid configuration."));
```

Comment:
As the `ThingStatusInfo` is translatable I am wondering if we should introduce a custom translation key for this description (e.g.`@text/missing-or-invalid-configuration`).

Reply:
Where Do I have to place the correct value then? `ThingManagerImpl.properties` in the i81n-folder? Or `ThingStatusInfo.properties`?

Reply:
I think it should be place in a new properties file with ame `SystemThingStatusInfos.properties`.